media-queries-exercise/firat/global.css

```diff
@@ -7,29 +7,65 @@ body {
   background-color: #f1f2f3;
   font-family: Arial, sans-serif;
   margin: 0;
+  font-size: small;
 }
 
 .header {
+  grid-area: header;
   background-color: #006c67;
 }
 .sidebar-left {
+  grid-area: sidebar-left;
   background-color: #003844;
 }
 .sidebar-right {
+  grid-area: sidebar-right;
   background-color: #ffb100;
 }
 .content {
+  grid-area: content;
   background-color: #ffebc6;
 }
 .footer {
   background-color: #006c67;
+  grid-area: footer;
 }
 
 .content > div {
   height: 300px;
 }
 
 .container {
+  display: grid;
+  grid-template-columns: 150px 1fr 150px;
+  grid-template-rows: 80px 1fr 150px;
   width: 100vw;
   height: 100vh;
-}
\ No newline at end of file
+  grid-template-areas:
+    "header header header"
+    "sidebar-left content sidebar-right"
+    "sidebar-left footer footer";
+}
+
+@media all (max-width: 768px) {
```

Comment:
Side bars are still visible on devices up to 768px wide. Try adding `display: none;` for `.sidebar-left`and `.sidebar-right`.